src/main/resources/schemas/nhinc/common/configAdmin.xsd

```diff
@@ -28,73 +28,81 @@
         <xsd:sequence>
             <xsd:element name="editCertRequest" type="tns:EditCertificateRequestType" minOccurs="1" maxOccurs="1" />
             <xsd:element name="configAssertion" type="tns:ConfigAssertionType" minOccurs="1" maxOccurs="1" />
+			<xsd:element name="isTrustStore" type="xsd:boolean" minOccurs="1" maxOccurs="1" default="true" />
         </xsd:sequence>
     </xsd:complexType>
     <xsd:element name="EditCertificateRequestMessage" type="tns:EditCertificateRequestMessageType" />
     
     <xsd:complexType name="EditCertificateRequestType">
         <xsd:sequence>
-            <xsd:element name="alias" type="xsd:string" minOccurs="1" maxOccurs="1" />
+            <xsd:element name="oldAlias" type="xsd:string" minOccurs="1" maxOccurs="1" />
+			<xsd:element name="newAlias" type="xsd:string" minOccurs="1" maxOccurs="1" />
+			<xsd:element name="certData" type="xsd:base64Binary" xmime:expectedContentTypes="application/octet-stream"  minOccurs="1" maxOccurs="1" />
         </xsd:sequence>
     </xsd:complexType>
     <xsd:element name="EditCertificateRequest" type="tns:EditCertificateRequestType" />
-        
-    <xsd:complexType name="EditCertificateResponseMessageType">
-        <xsd:sequence>
-            <xsd:element name="status" type="xsd:boolean"/>
-			<xsd:element name="message" type="xsd:string"/>
-        </xsd:sequence>
-    </xsd:complexType>
-    <xsd:element name="EditCertificateResponseMessage" type="tns:EditCertificateResponseMessageType"/>
+    
+    <xsd:element name="EditCertificateResponseMessage" type="tns:SimpleResponseMessageType"/>
     
 	 <xsd:complexType name="DeleteCertificateRequestMessageType" >
         <xsd:sequence>
-            <xsd:element name="deleteCertRequest" type="tns:DeleteCertificateRequestType" minOccurs="1" maxOccurs="1" />
+            <xsd:element name="alias" type="xsd:string" minOccurs="1" maxOccurs="1" />
             <xsd:element name="configAssertion" type="tns:ConfigAssertionType" minOccurs="1" maxOccurs="1" />
         </xsd:sequence>
     </xsd:complexType>
     <xsd:element name="DeleteCertificateRequestMessage" type="tns:DeleteCertificateRequestMessageType" />
     
-    <xsd:complexType name="DeleteCertificateRequestType">
-        <xsd:sequence>
-            <xsd:element name="alias" type="xsd:string" minOccurs="1" maxOccurs="1" />
-        </xsd:sequence>
-    </xsd:complexType>
-    <xsd:element name="DeleteCertificateRequest" type="tns:DeleteCertificateRequestType" />
-        
-    <xsd:complexType name="DeleteCertificateResponseMessageType">
-        <xsd:sequence>
-            <xsd:element name="status" type="xsd:boolean"/>
-			<xsd:element name="message" type="xsd:string"/>
-        </xsd:sequence>
-    </xsd:complexType>
-    <xsd:element name="DeleteCertificateResponseMessage" type="tns:DeleteCertificateResponseMessageType"/>
+    <xsd:element name="DeleteCertificateResponseMessage" type="tns:SimpleResponseMessageType"/>
```

Comment:
I don't think these subtypes are necessary to the simple type.  Just use the Simpletype response directly in the wsdl.